Packs/Sixgill-Darkfeed/Integrations/Sixgill_Darkfeed/Sixgill_Darkfeed_test.py

```diff
@@ -531,3 +523,32 @@ def test_get_indicators_command(mocker):
     submitted_indicators = 0
 
     assert output[2] == expected_ioc_output
+
+
+def test_feed_tags(mocker):
+    """
+    Given:
+    - tags parameters
+    When:
+    - Executing fetch command on feed
+    Then:
+    - Validate the tags supplied are added to the tags list in addition to the tags that were there before
+    """
+    global bundle_index
+    global submitted_indicators
+
+    mocker.patch.object(demisto, 'params', return_value=init_params())
+    mocker.patch('requests.sessions.Session.send', new=mocked_request)
+
+    from Sixgill_Darkfeed import fetch_indicators_command
+    from sixgill.sixgill_feed_client import SixgillFeedClient
+    from sixgill.sixgill_constants import FeedStream
+
+    client = SixgillFeedClient("client_id",
+                               "client_secret",
+                               "some_channel",
+                               FeedStream.DARKFEED,
+                               demisto, 1000)
+
+    output = fetch_indicators_command(client, tags=['tag1', 'tag2'])
+    assert all(item in output[0]['fields']['tags'] for item in ['tag1', 'tag2'])
```

Comment:
does some of the indicators contain yet another tag? e.g: compromised,malware hash, shell
please ass a test case taht validates that the `union` works as expected and both `feedTags` and the IOC `labels` are present. in an indicator of your choice.

Reply:
Yes they all contain a tag and the test checks that the 'tag1' and 'tag2' are added